libs/cart/src/drivers/interfaces/cart-shipping-information-service.interface.ts

```diff
@@ -0,0 +1,30 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { DaffCartShippingRate } from '../../models/cart-shipping-rate';
+import { DaffCart } from '../../models/cart';
+
+/**
+ * The interface responsible for mediating the interaction of the shipping
+ * information of a cart with a given platform.
+ */
+export interface DaffCartShippingInformationServiceInterface<T extends DaffCartShippingRate>{
```

Comment:
Should the generic extend from `DaffCartShippingInformation` instead?

Reply:
I don't know what this type really means yet, so I'm electing for the less explicit type to start.

Reply:
I don't what this type truly looks like yet, so I'm opting for the less explicit type.

Reply:
Got it, I think that's fine for now.